### DISCUSSION TITLE
Question about canvas support

### DISCUSSION THREAD

**Discussion by amperag** [oleeskild/obsidian-digital-garden#121] [General]
Hey @oleeskild, are you thinking about supporting the new "canvas" plugin which is currently in insider testing?

https://forum.obsidian.md/t/obsidian-release-v1-1-0-insider-build/48954

**Comment by oleeskild** [comment]
Hi @amperag 

TL;DR: I will have to wait until they publish an API to see if it is possible.

I will into it as soon as they make an API for it available. I suspect it would be hard to support, as it is using a custom JSON format for saving the canvas. As regular notes are simply formatted as markdown, I was able to use existing tools for the HTML render.   
To render the canvas however, I would probably need to reimplement their render engine for it to work properly. Which is not something I'm gonna do. 😅

But hopefully they will provide an API that lets us export it as some standard format, like an SVG or something. Then it shouldn't be too hard. We'll have to wait and see.

**Reply by amperag** [reply]
Alright thanks for the diligent answer @oleeskild! In that case let's hope the obsidian Devs provide something you can use. 🙂

**Reply by createperhaps** [reply]
Hi its been three years, any updates?
